scripts/debug

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash -e
+
+# This script is based on raspinfo: https://github.com/raspberrypi/utils/blob/master/raspinfo/raspinfo
+# raspibfo is Copyright: 2018-2019, Raspberry Pi (Trading) Ltd.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+# 3. Neither the name of the University nor the names of its contributors
+#    may be used to endorse or promote products derived from this software
+#    without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT 
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR 
+# A PARTICULAR PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE HOLDERS OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, 
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, 
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR 
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF 
+# LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING 
+# NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS 
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# This fork is Copyright: 2020 Umbrel.
+
+[[ -f "/etc/default/umbrel" ]] && source "/etc/default/umbrel"
```

Comment:
We can drop this since the file is already [sourced in `/etc/profile`](https://github.com/getumbrel/umbrel-os/blob/master/stage2/04-set-umbrel-os-version/00-patches/01-profile.diff#L11).

```suggestion
```

Reply:
@mayankchhabra Can this then also be removed [in the update scripts](https://github.com/getumbrel/umbrel/blob/master/scripts/update/01-run.sh#L20)?

Reply:
Update scripts run as sudo and `/etc/profile` isn't automatically sourced for sudo :(

Reply:
@mayankchhabra Both users that used the script in the telegram group yet have had the same issue: the script didn't detect Umbrel OS. Should I add it again?

Reply:
Maybe you need to add an "export " to the file?

Reply:
Good spot, Aaron! That is exactly it.

Let's go with sourcing the file for now and then eventually we'll update the `/etc/default/umbrel` file with an export in an OTA update.

Reply:
Good spot, Aaron! That is exactly it.

Let's go with sourcing the file for now and then eventually we'll update the /etc/default/umbrel file with an export in an OTA update.